modules/python-modules/syslogng/modules/s3/s3_destination.py

```diff
@@ -182,8 +182,7 @@ def generate_persist_name(options: Dict[str, Any]) -> str:
         return f"s3({','.join([options['url'], options['bucket'], str(options['object_key'])])})"
 
     def __load_persist(self) -> None:
-        for path_str in glob(pathname="*.json", root_dir=self.working_dir):
-            path = Path(self.working_dir, path_str)
+        for path in Path(self.working_dir).glob("*.json"):
```

Comment:
remove the no longer needed import glob from glob line (L35) since it no longer needed

Reply:
otherwise lgtm